addons/website_slides/static/src/js/tours/slides_tour.js

```diff
@@ -46,7 +46,7 @@ wTourUtils.registerWebsitePreviewTour('slides_tour', {
     position: 'bottom',
     width: 260,
 }, {
-    trigger: ':iframe a.btn-primary.o_wslides_js_slide_upload',
+    trigger: '.o_iframe[is-ready="true"]:iframe a.btn-primary.o_wslides_js_slide_upload',
```

Comment:
Should we not use waitIframeIsReady before this step instead? It looks like it is one of the changes of the commit you referenced.

Same for other trigger :)

Reply:
Hello @Odoonan 
I tried to use that function but still, the tour was failing
so I have used this selector :smiley: 
Thanks!!

Reply:
I am not sure here. It feels strange that we cannot use the waitIframeIsReady, as the fix is very similar to what is done inside that function... All tours using it are 'test: true', which is not the case here. Maybe this is why it won't work properly. I'm out of ideas, so I will green but leave this conversation open, maybe reviewers will find a better solution.

Reply:
Okay @Odoonan 
Actually, I was also struggling to find the solution so 
this fix was suggested by Boi Bruno (boi)

Reply:
Hello :wave: 

Have to say I agree with @Odoonan here, it feels strange not to use the tools provided.
Did you try putting the "waitIframeIsReady" before the steps accessing the modal element? It seems this step is not the first one, which is even weirder.

@brboi Would you please elaborate on this choice? So that we understand what's going on exactly.

Reply:
Ah I see, so we are basically inside an exception case here that requires a specific selector.

Let's go for this solution then, no need to spend too much time on a technical detail inside a tour.

Thanks for the detailed explanations!